yklua/README.md

```diff
@@ -17,3 +17,7 @@ $ /path/to/yklua/src/lua ./fannkuchredux.lua 3
 $ /path/to/yklua/src/lua ./fasta.lua 10
 $ /path/to/yklua/src/lua ./binary-trees.lua 
 ```
+
+## License
+
+[Revised BSD license](./LICENSE) applies to all the files located in yklua directory.
```

Comment:
s/yklua/this/

Reply:
took me a while but I got it :point_right: https://github.com/ykjit/ykbench/pull/1/commits/9064876d3c17b02531680ff579c706bda9631caa